ci/priv-integration.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+# Assumes that the current environment is a privileged container
+# with the host mounted at /run/host.  We can basically write
```

Comment:
This clearly works as is, but it isn't very clear to me why this bindmount was introduced. We are only using `ostree` and `ostree-ext-cli`, plus I don't expect the Ubuntu host to have an `/ostree` directory. Is this equivalent to an empty-directory bindmount? Is it to avoid some kind of quota limit?

Reply:
> plus I don't expect the Ubuntu host to have an /ostree directory. 

We're making one here!  That's a nice thing about ostree, we can just install inside an existing system.

Reply:
Basically, we want to try to install onto the real host filesystem, we don't want to operate on the container `overlayfs`.